ur_robot_driver/package.xml

```diff
@@ -22,6 +22,7 @@
 
   <build_depend>boost</build_depend>
   <build_depend>industrial_robot_status_interface</build_depend>
+  <build_depend>rostest</build_depend>
```

Comment:
Should this be a `test_depend`?

Reply:
According to http://wiki.ros.org/unittest#Update_your_manifest and http://wiki.ros.org/Quality/Tutorials/UnitTesting#Main_testing_tools_in_ros it is a build_depend. However, http://docs.ros.org/kinetic/api/catkin/html/howto/format2/rostest_configuration.html states that it should be a build depend.

Reply:
I believe you mean to wrote:

> However, http://docs.ros.org/kinetic/api/catkin/html/howto/format2/rostest_configuration.html states that it should be a *test depend*.

See also [Is test_depend actually used for anything?](https://answers.ros.org/question/115526) on ROS Answers.

Reply:
Thanks, that explains that :-)